x-pack/plugins/ml/public/application/trained_models/models_management/pipelines/expended_row.tsx

```diff
@@ -49,23 +58,57 @@ export const ProcessorsStats: FC<ProcessorsStatsProps> = ({ stats }) => {
     },
     {
       field: 'stats.count',
-      name: i18n.translate(
-        'xpack.ml.trainedModels.modelsList.pipelines.processorStats.countHeader',
-        {
-          defaultMessage: 'Count',
-        }
+      name: (
+        <EuiFlexGroup gutterSize="xs">
+          <EuiFlexItem grow={false}>
+            <FormattedMessage
+              id="xpack.ml.trainedModels.modelsList.pipelines.processorStats.countHeader"
+              defaultMessage="Count"
+            />
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiToolTip
+              position="top"
+              content={
+                <FormattedMessage
+                  id="xpack.ml.trainedModels.modelsList.pipelines.processorStats.countDescription"
+                  defaultMessage="Total number of documents ingested during the lifetime of this node"
+                />
+              }
+            >
+              <EuiIcon tabIndex={0} type="iInCircle" color={'subdued'} />
```

Comment:
For consistency with the help icons we have on the anomalies table columns, these should be:
```
              <EuiIcon
                tabIndex={0}
                type="questionInCircle"
                className="eui-alignTop"
                size="s"
                color={'subdued'}
              />
```

![image](https://user-images.githubusercontent.com/7405507/142026864-8305c8de-9c6d-4453-9efd-b1d5f7814c83.png)

Reply:
Done in b6c34475ea4cecfdb633eef14f02e406e0daff3f